user-guide/Advanced_Modules/Dashboards_and_Low_Code_Apps/Low_Code_Apps/Low-Code_Apps_configuration/LowCodeApps_Custom_Icon.md

```diff
@@ -15,3 +15,6 @@ You can then:
 - Use the color picker to specify a different color for the app.
 
 To close the color and icon editor, click the icon in the top-left corner again.
+
+> [!NOTE]
+> You can also change the theme for a specific page within your app in the same way as in the Dashboards app. These themes are in fact shared with the Dashboards app. Note also that it is currently not yet possible to modify an existing theme directly in the Low-Code Apps, but you can do to via the Dashboards app. For more information, see [Configuring the dashboard layout](xref:Configuring_the_dashboard_layout).
```

Comment:
"but you can do to via the Dashboards app." should probably be "but you can do so via the Dashboards app."

Reply:
Corrected!